src/main/scala/dsptools/numbers/algebra_types/helpers/Sign.scala

```diff
@@ -69,6 +69,7 @@ object Sign {
     override def sign(a: Sign): Sign = a
     def signum(a: Sign): ComparisonBundle = ComparisonHelper(a.zero, a.neg)
     def abs(a: Sign): Sign = if (a == Negative) Positive else a
+    def context_abs(a: Sign): Sign = if (a == Negative) Positive else a
```

Comment:
As currently implemented not sure how useful it is. I think I left it in at the time wondering if it would be useful if you wanted to use signed-magnitude notation (e.g. ldpc). I won't be mad if sign just gets ripped out. Maybe that should be in a separate PR though